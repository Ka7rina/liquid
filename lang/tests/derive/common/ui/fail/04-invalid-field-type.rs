use liquid::InOut;
use liquid_lang as liquid;

#[derive(InOut)]
pub struct MyInOut {
    b: bool,
    i: f32,
}

fn main() {}
