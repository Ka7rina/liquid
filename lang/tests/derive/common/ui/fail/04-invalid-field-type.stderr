error[E0277]: the trait bound `f32: You_Should_Use_An_Valid_Field_Type` is not satisfied
 --> $DIR/04-invalid-field-type.rs:7:8
  |
7 |     i: f32,
  |        ^^^ the trait `You_Should_Use_An_Valid_Field_Type` is not implemented for `f32`
